index.js

```diff
@@ -154,6 +154,10 @@ function applyStyle() {
 	const argsLen = args.length;
 	let str = argsLen !== 0 && String(arguments[0]);
```

Comment:
Yeah should be fine. `String(arguments[0])` is going to evaluate to `String(undefined)`, which will just evaluate to `"undefined"` - that is, in the case that `argsLen === 0`.